cpp/include/cudf/io/orc.hpp

```diff
@@ -445,24 +445,24 @@ class orc_writer_options {
   /**
    * @brief Whether writing column statistics is enabled/disabled.
    */
-  bool enable_statistics() const { return _enable_statistics; }
+  bool is_enabled_statistics() const { return _enable_statistics; }
```

Comment:
Not sure which one is better, `is_enabled_statistics` or `is_statistics_enabled`?

Reply:
There are other binary options, and they use `is_enabled_<feature name>`. AFAICT, this change aligns this name with other interfaces.